docs/zh/latest/plugins/openid-connect.md

```diff
@@ -59,7 +62,7 @@ OAuth 2 / Open ID Connect（OIDC）插件为 APISIX 提供身份验证和自省
 ![token introspection](../../../assets/images/plugin/oauth-1.png)
 
 以下是 curl 命令，用于将插件启用到外部服务。
-通过自省请求标头中提供的令牌，此路由将保护 https://httpbin.org/get（echo 服务）。
+通过自省请求标头中提供的令牌，此路由将保护 https://httpbin.org/get (echo 服务）。
```

Comment:
it should be a fullwidth bracket and a space, but not changed to halfwidth

Reply:
just updated